c2rust-analyze/src/analyze.rs

```diff
@@ -1095,6 +1101,29 @@ fn run(tcx: TyCtxt) {
         );
     }
 
+    // For testing, putting #[c2rust_analyze_test::force_non_null_args] on a function marks its
```

Comment:
This function is already over 700 lines, could we move this step into a separate function?

Reply:
Moved this and the adjacent bit of `TestAttr` handling into their own helper functions: 8851c89.  As we continue working on this code, we can do more refactoring as we go.